mrp_bom_version/views/mrp_bom_view.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.actions.act_window" id="mrp.mrp_bom_form_action">
+            <field name="domain">['|',('active','=',True),('active','=',False)]</field>
+        </record>
+
+        <record id="mrp.mrp_bom_tree_parent_view" model="ir.ui.view">
+            <field name="name">mrp.bom.tree</field>
+            <field name="model">mrp.bom</field>
+            <field name="arch" type="xml">
+                <tree>
+                    <field name="name"/>
+                    <field name="code"/>
+                    <field name="sequence"/>
+                </tree>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="mrp_bom_version_tree_view">
+            <field name="name">mrp.bom.version.tree</field>
+            <field name="model">mrp.bom</field>
+            <field name="inherit_id" ref="mrp.mrp_bom_tree_parent_view"/>
+            <field name="arch" type="xml">
+                <field name="name" position="after">
+                    <field name="active" />
+                    <field name="version" />
+                    <field name="state" />
+                    <field name="historical_date"/>
+                    <button name="button_draft" type="object" string="Draft"
+                            groups="mrp_bom_version.group_mrp_bom_version"
+                            attrs="{'invisible':[('state','!=','active')]}"
+                            icon="terp-document-new" />
+                    <button name="button_activate" type="object" string="Activate"
+                            attrs="{'invisible':[('state','not in',(False, 'draft'))]}"
+                            icon="terp-camera_test"
+                            confirm="You will activate the BoM. If you haven't set a route yet, then you won't be able to do it after this. Are you sure you want to proceed?"/>
```

Comment:
You will now activate the BoM. Please make sure to set up a route as after activation, then you will not be allowed to do so. Are you sure you want to proceed?